### ISSUE TITLE
Have a way to clear bot messages from chat view

### ISSUE THREAD

**Issue by kotp** [seveneightn9ne/keybase-reminder-bot#43]
After a while, the messages from reminder bot does not need to be in the chat view.  It would be nice to instruct the bot to delete its own messages from the chat history.  Perhaps in categories: `!delete help feedback` or `!delete reminder messages up to yesterday`

Also, setting explode timer, so they will explode at a configured time.

**Comment by seveneightn9ne** [comment]
In the Keybase chat settings, you can set the chat to auto-delete messages after some time or one-off clear the entire conversation. Would that solve the same thing? Or do you only want to clear reminderbot's history and not the rest of the chat.

For the exploding timer, I think it would be good if reminderbot replied back with the same exploding message settings from the user's message.

**Comment by kotp** [comment]
> Or do you only want to clear reminderbot's history and not the rest of the chat.

Yes, that... general chat room retention is a different thing, so yes, I am asking about getting rid of stale bot messages.  The serve, after a while, only to clutter the meaningful context of the room.

> For the exploding timer, I think it would be good if reminderbot replied back with the same exploding message settings from the user's message.

That sounds good, but if I forget to set the timer for my triggering message, I am stuck with not being able to delete the bot message.

Perhaps this can be addressed by a trigger `!reminderbot delete messages after 15 minutes ago` for example.

**Comment by seveneightn9ne** [comment]
Ok, I understand, this totally makes sense. Maybe something like `!clear history` to wipe all the messages, or do you think it's important to be able to not delete the most recent messages?

**Comment by kotp** [comment]
I think I would want to leave some recent for a while, especially if in a team setting.

**Comment by seveneightn9ne** [comment]
Oh gotcha, you mean like a permanent setting that reminderbot would delete old messages. Makes sense.

**Comment by kotp** [comment]
Configurable, as I am pretty sure my wants aren't everyone's wants.
